uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/Service/LeaveRequestRightsTest.php

```diff
@@ -281,6 +282,38 @@ public function testCanCancelToilWithPastDatesReturnsTrueWhenAbsenceTypeAllowsPa
     $this->assertTrue($leaveRightsService->canCancelToilWithPastDates($this->leaveContact, $absenceType->id));
   }
 
+  public function testGetLeaveContactsCurrentUserHasAccessToWhenUserIsStaff() {
```

Comment:
What is a "Leave Contact"? What is the difference between that and a regular Contact?

Also, for the test name, it would be better to use to describe what is the expected behaviour instead of naming the scenario. For example, this test could be named something like "testStaffMembersShouldOnlyHaveAccessToThemselves". The same apply for the other tests